introduction/architectural-overview.md

```diff
@@ -1 +1,35 @@
 # Architectural overview
+
+## Tiers
+Valtimo is a based on a three-tier architecture. 
+
+-	The presentation tier is the user interface, based on Angular. It’s referred to as Frontend. 
+-	The (business) logic tier is the heart of Valtimo, built in Kotlin/Java. It’s referred to as Backend.
+-	For the data tier PostgreSQL is used – MySQL is supported for versions < 9.
+
+## Product v.s. implementation
+
+An installation of the product is called an Implementation. In the backend an implementation is a Spring boot project. 
+ 
+## Using a code repository & buildpipeline for building business processes
+
+Developers can add models, configuration and code in the implementation. Models (DMN, BPMN in XML) and configuration (in JSON) are loaded in the database during startup. Working with a repository and CI/CD pipeline has the following advantages: 
+
+-	Models and configuration are versioned in the repository. In this way, changes are tracked and quality can be improved by adding review processes. 
```

Comment:
JSON configuration is usually not versioned

Reply:
I meant the versioning of Git - as in being able to track changes.